sqlglot/dialects/dremio.py

```diff
@@ -142,8 +153,22 @@ class Parser(parser.Parser):
             "DATE_ADD": build_date_delta_with_cast_interval(exp.DateAdd),
             "DATE_SUB": build_date_delta_with_cast_interval(exp.DateSub),
             "ARRAY_GENERATE_RANGE": exp.GenerateSeries.from_arg_list,
+            "CURRENT_DATE_UTC": lambda args: Dremio._current_date_utc_expr(),
+        }
+
+        KEYWORDS = {
+            **tokens.Tokenizer.KEYWORDS,
+            "CURRENT_DATE_UTC": TokenType.COMMAND,
         }
```

Comment:
This doesn't seem right; why was it needed? `COMMAND` is a token type reserved for statements that we want to wrap in dummy AST nodes to avoid having the parser fail when it encounters them.